docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,319 @@
+# Mobile exercises
+
+Writing a DataCamp for Mobile unique. The upside of the mobile platform is that
+you can reach a much broader audience because the convenience is higher and the
+barrier to entry is lower. The challenge is that there are many more constraints
+to work within. Primary among these are the restricted screen real estate and
+the lack of an interactive interpreter. These constraints require you to think
+carefully about the way you introduce concepts and the examples that you use.
+
+# Basic exercise structure
+
+There are roughly 5 exercise types available to you as an author on DataCamp
+Mobile:
+
+* [Select Output](select-output.md)
```

Comment:
One thing that isn't super-clear is how the exercise type is chosen, since there is no `exercise_type` element in the YAML. I presume it's auto-determined from the type of distractor block. Since this is a little unintuitive, best to state this.

Reply:
I've highlighted this fact now.